ccd-definition/ComplexTypes/JudgeAndLegalAdvisor.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "JudgeAndLegalAdvisor",
+    "ListElementCode": "judgeTitle",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "JudgeOrMagistrateTitle",
+    "ElementLabel": "Judge or magistrate's title",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "JudgeAndLegalAdvisor",
+    "ListElementCode": "judgeLastName",
+    "FieldType": "Text",
+    "ElementLabel": "Last name",
+    "FieldShowCondition": "judgeTitle=\"HER_HONOUR_JUDGE\" OR judgeTitle=\"HIS_HONOUR_JUDGE\" OR judgeTitle=\"DEPUTY_DISTRICT_JUDGE\"",
```

Comment:
I'd thought the same but because I'm using 'FixedRadioList' result is a string rather than an array. Contains does not seem to help. Will continue looking into it.

Reply:
No luck with Contains or judgeTitle="*Judge".